.github/issue_template/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Melkeydev Discord
```

Comment:
@Melkeydev This redirects people to the community, not sure if this is what you want of the description. Please met me know before merge.